package.json

```diff
@@ -42,7 +42,8 @@
         "cy:run": "start-server-and-test 'yarn cy:start' http://localhost:3000 'yarn cypress run --browser chrome headless'"
     },
     "devDependencies": {
-        "@badeball/cypress-cucumber-preprocessor": "^20.1.0",
+        "@babel/plugin-syntax-dynamic-import": "^7.8.3",
```

Comment:
Why is this dependency added?

Reply:
After I upgraded and added some some dev-dependencies, I cleared my `node_modules` and ran `npx yarn-deduplicate`. This resulted in a significant amount of changes in the `yarn.lock` and the package `@dhis2/cli-app-scripts` started complaining about this missing dependency, so I just added it. Since this was a change in the dev dependencies only I didn't worry about it too much. Do you think we need to investigate it further.

Reply:
No, I don't think so. But maybe just add a short comment to the PR description.